metagenomics.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+import argparse
+import gzip
+import shutil
+import util.file
+import tools.kraken
+import tools.krona
+import tools.diamond
+__commands__ = []
+
+
+def kraken(inFastq1, inFastq2, db, outReport=None, outReads=None,
```

Comment:
As a general UI rule, we've been using exclusively BAM files as inputs and outputs for viral-ngs top-level scripts. Fastqs are always hidden, ephemeral / never persist. Can you modify this to take a single BAM as input and start off the method with a conversion as done [elsewhere](https://github.com/broadinstitute/viral-ngs/blob/master/taxon_filter.py#L271)? Same for the krona method.